packages/app-store/routing-forms/components/react-awesome-query-builder/config/config.tsx

```diff
@@ -1,11 +1,19 @@
 import type { ChangeEvent } from "react";
-import type { Settings, Widgets, SelectWidgetProps, SelectWidget } from "react-awesome-query-builder";
+import type {
+  Settings,
+  Widgets,
+  SelectWidgetProps,
+  SelectWidget as SelectWidgetType,
```

Comment:
SelectWidget was also a value defined on line 26. So, renamed the type to fix the prettier crash